website/docs/r/build_folder_permissions.html.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_build_folder_permissions"
+description: |-
+  Manages permissions for a AzureDevOps Build Folder
+---
+
+# azuredevops_build_folder_permissions
+
+Manages permissions for a Build Folder
+
+~> **Note** Permissions can be assigned to group principals and not to single user principals.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "example" {
+  name               = "Example Project"
+  work_item_template = "Agile"
+  version_control    = "Git"
+  visibility         = "private"
+  description        = "Managed by Terraform"
+}
+
+data "azuredevops_group" "example-readers" {
+  project_id = azuredevops_project.example.id
+  name       = "Readers"
+}
+
+resource "azuredevops_build_folder" "example" {
+  project_id  = azuredevops_project.example.id
+  path        = "\\ExampleFolder"
+  description = "ExampleFolder description"
+}
+
+resource "azuredevops_build_folder_permissions" "example" {
```

Comment:
The example `azuredevops_build_folder_permissions.example`  missing the path configuration `path        = azuredevops_build_folder.example.path`